content/docs/replication/deployment/powerflex.md

```diff
@@ -0,0 +1,255 @@
+---
+title: PowerFlex
+linktitle: PowerFlex
+weight: 6
+description: Enabling Replication feature for CSI PowerFlex
```

Comment:
Please update the release notes section about new feature.
https://dell.github.io/csm-docs/docs/replication/release/#new-featureschanges